lib/tasks/publish_deal.rake

```diff
@@ -0,0 +1,16 @@
+desc "Publish Deal Everyday at 10 am"
+task :publish_deal => :environment do
+  @publishing_deal = Deal.to_publish
+  @unpublishing_deal = Deal.to_unpublish
+  count = 0
+  @publishing_deal.each do |deal|
+    if count < 2 && deal.update(published_at: Time.current)
+      count += 1
+    else
+      deal.update_column(:publish_at, Time.current + TWENTY_FOUR_HOURS)
```

Comment:
why update_column and not update?

Reply:
Because there is a validation in deal model which restrict updating ```publish_at``` if publish_at - Time.current is less than 24 hours. so to skip that validation , I used update_column